packages/@sfx/search-plugin/src/search-plugin.ts

```diff
@@ -0,0 +1,63 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { BridgeConfig, BrowserBridge, Query } from 'groupby-api';
+
+/**
+ * This plugin exposes an instance of the GroupBy search client.
+ */
+export default class SearchPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'search',
+      depends: [],
+    };
+  }
+
+  /**
+   * The values that the Search Data Source plugin expose to the Core entity.
+   */
+  browserBridge: SearchPluginExposedValue;
+  options: SearchPluginOptions;
```

Comment:
We don't need to be storing options since we won't be using it after we construct the BrowserBridge. Please remove.